extension/manifest.json

```diff
@@ -39,7 +39,7 @@
       "matches": [
           "<all_urls>"
       ],
-      "js": ["js/add_input_icon.js", "js/fill_relay_address.js"],
+      "js": ["js/installation_indicator.js", "js/add_input_icon.js", "js/fill_relay_address.js"],
```

Comment:
We don't need to run this on `<all_urls>` - can we move it to a `"matches": [ "http://127.0.0.1/*" ]` item?